src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -59,6 +60,18 @@ public static void WriteMetricsCollection(this PrometheusExporter exporter, Stre
                     {
                         WriteSum(writer, builder, metric.Attributes, (metric as ISumMetricDouble).DoubleSum);
                     }
+                    else if (metric.MetricType == MetricType.DoubleGauge)
+                    {
+                        var gaugeMetric = metric as IGaugeMetric;
+                        var doubleValue = (double)gaugeMetric.LastValue.Value;
+                        WriteGauge(writer, builder, metric.Attributes, doubleValue);
+                    }
+                    else if (metric.MetricType == MetricType.LongGauge)
+                    {
+                        var gaugeMetric = metric as IGaugeMetric;
+                        var longValue = (long)gaugeMetric.LastValue.Value;
```

Comment:
Should we skip this for now then? We could just have the same logic as MetricType.DoubleGauge.

Reply:
I'll mark a TODO here to comeback to this, in the my next PR (which adds histogram)